.pre-commit-config.yaml

```diff
@@ -43,11 +43,22 @@ repos:
   hooks:
   - id: mypy
 
-- repo: local
+- repo: git://github.com/pre-commit/mirrors-pylint.git
+  rev: v2.3.1
   hooks:
   - id: pylint
-    language: system
-    name: PyLint
-    files: \.py$
-    entry: python3 -m pylint.__main__
-    args: []
+    args:
+    - --msg-template={path}:L {line:3d}:({symbol}) {msg} (C {column:2d}), ::{msg_id}
+    - --output-format=colorized
+    - |-
+        --init-hook=
+        import os, platform, sys;
+        sys.path.insert(
+            0,
+            "".join([os.getenv("HOME"), "/virtualenv/python", platform.python_version(),
```

Comment:
I do not clear understand. What do I have to do?

Reply:
Revert this change. Unfortunately, you cannot do this via `git revert` because it's not in a separate atomic commit. So you'd have to do smth like `git checkout HEAD -- .pre-commit-config.yaml`. Use atomic commits to make your life easier.